cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -77,6 +77,9 @@
     public static final long DEFAULT_CONNECT_BUILD_TIMEOUT_MS = 300_000;
     public static final int DEFAULT_STRIMZI_OPERATIONS_THREAD_POOL_SIZE = 10;
 
+    //Hostname as injected by K8s is used to identify an "instance" of a controller/operator, needed for K8s events
+    public static final String CLUSTER_OPERATOR_HOST = "HOSTNAME";
```

Comment:
I think this is confusing ...
* Value is `HOSTNAME`
* Constant is `CLUSTER_OPERATOR_HOST`
* And the instance value is `operatorInstance`

Please decide what are you configuring. Is it a hostname? Is it some instance ID? Now it is not clear what it is. The environment variable should be maybe also prefixed with `STRIMZI_` as all other options?

Reply:
Yep, good point, it would be better to explicitly inject it from the metadata like we do for the namespace, and prefix it as you suggest.